Packs/Stairwell/Integrations/Inception/Inception.py

```diff
@@ -60,20 +60,20 @@ def file_enrichment_command(client, file_hash):
                 md += f'Filename(s): {filenames_string}\n'
 
             # Count the number of assets if there are seen assets
-            if "assets" in response:
+            if responseJson['data']['attributes']['inception']['assets']:
                 seen_assets = len(responseJson['data']['attributes']['inception']['assets'])
```

Comment:
I see all the conditions used to check if the field is present in the dictionary, are we sure we want to remove this check? if the field is not present this will create an error.
If it's possible that the field might not exist maybe we should change the condition to-
           ` if "assets" in response and responseJson['data']['attributes']['inception']['assets']:`
What do you think?

Reply:
Thanks for the note @ostolero! This PR is adding support for the fact our API will **always** respond with the fields, but will be blank if we have no results. So in your example 'assets' will always exist, but may be empty, which is what we're testing for.

Reply:
I see great then this should work 👍